Objects/tupleobject.c

```diff
@@ -832,6 +832,7 @@ static PyMethodDef tuple_methods[] = {
     TUPLE___GETNEWARGS___METHODDEF
     TUPLE_INDEX_METHODDEF
     TUPLE_COUNT_METHODDEF
```

Comment:
Since this method is going to become so pervasive, consider adding a tp_slot for it.

Reply:
Hm, I don't know if it's worth it. There are currently just 7 occurrences in Object/ and another 7 in Modules/.

Reply:
It doesn't have to be decided now, you can switch to slots in the future if the balance tips the other way.